<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPhones</class>
 <widget class="QWidget" name="WizardPhones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="wizardTittleLabel">
     <property name="text">
      <string>Enter your phone number(s): </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>156</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QFrame" name="framePhone1">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mobile</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="framePhone2">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Home  </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QFrame" name="framePhone3">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Work  </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QFrame" name="framePhone4">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fax   </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QFrame" name="framePhone5">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Other </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>156</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
